c/cmake/BLAKE3/Utils.cmake

```diff
@@ -0,0 +1,21 @@
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.10)
+  include_guard(GLOBAL)
+endif()
+
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
+  set(BLAKE3_CXX_COMPILER_FRONTEND_VARIANT ${CMAKE_CXX_COMPILER_FRONTEND_VARIANT})
```

Comment:
```suggestion
  set(BLAKE3_CXX_COMPILER_FRONTEND_VARIANT "${CMAKE_CXX_COMPILER_FRONTEND_VARIANT}")
```

Reply:
Unfortunately this PR is already referenced in the downstream patches to fix the BLAKE3 build and the PR contents is hashed in the package.

So I'll have to open a new PR with these changes: https://github.com/BLAKE3-team/BLAKE3/pull/483